knowledge-base/how-to-refer-telerik-reporting-assemblies-in-version-control-repository-with-limited-number-of-licenses.md

```diff
@@ -0,0 +1,34 @@
+---
+title: How to refer Telerik Reporting assemblies in projects hosted in version control repository when only one/few developers hold licenses
+description: Solution won't compile to other developers in the team that are not licensed for Telerik Reporting
+type: how-to
+page_title: How to refer Telerik Reporting assemblies in version control repository with limited number of licenses
```

Comment:
If I am not mistaken, the tile has to be: How to Refer Telerik Reporting Assemblies in Version control Repository with Limited Number of Licenses